src/utils.ts

```diff
@@ -16,3 +17,29 @@ export const createMongoNoteInstance = async (
     }).save();
   }
 };
+
+const returnUrlParameters = (url: string) => {
+  const splitUrl = url.split("/");
+  const index = splitUrl.length;
+  return [splitUrl[index - 3], splitUrl[index - 2], splitUrl[index - 1]];
+};
+
+export const slackQuery = (request: Request, response: Response) => {
+  const { body } = request;
+  if (body.response_url) {
+    const [param1, param2, param3] = returnUrlParameters(body.response_url);
```

Comment:
I'd prefer that these names were more meaningful, but seeing as I know we're going to change them soon, don't worry about it.